From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Thu, 23 Apr 2026 16:59:36 +0800
Subject: Make sure file systems can be fixed on machines with locked root
 accounts (as Ubuntu does by default). Don't require --force for sulogin.

Bug-Debian: https://bugs.debian.org/326678
---
 login-utils/sulogin.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/login-utils/sulogin.c b/login-utils/sulogin.c
index e1b30de..075acba 100644
--- a/login-utils/sulogin.c
+++ b/login-utils/sulogin.c
@@ -1196,7 +1196,8 @@ int main(int argc, char **argv)
 				const char *passwd = pwd->pw_passwd;
 				char *answer;
 				int doshell = 0;
-				int deny = !opt_e && locked_account_password(pwd->pw_passwd);
+				/* Ubuntu's root account is locked by default allow access without --force */
+				int deny = 0;
 
 				doprompt(passwd, con, deny);
 
